tests/fuzzer/CMakeLists.txt

```diff
@@ -175,4 +175,16 @@ if(BUILD_DEBUGGER)
     scalar_debug.cpp
   )
   target_link_libraries(scalar_debug PRIVATE flatbuffers_nonfuzz)
+
+  add_executable(monster_debug
```

Comment:
if this is purely for people that debug fuzzer code, and not of interest to anyone else, should this be surrounded by some conditional?

e.g. in VS extra targets show up as projects, so if they're not of interest they probably shouldn't be there.

Reply:
It already surrounded with:
https://github.com/google/flatbuffers/blob/69b329fc87e7a10f84a0bd012e74e3af7549526d/tests/fuzzer/CMakeLists.txt#L146